### ISSUE TITLE
encoder training isn't stopping 

### ISSUE THREAD

**Issue by Sarajish** [CorentinJ/Real-Time-Voice-Cloning#1293]
i am trying to train encoder on a small dataset of 10 audios but it is taking months and not stopping. Can you please help me?

**Comment by CorentinJ** [comment]
Yeah it doesn't have a time limit, you have to stop it based on how it performs on the test set by looking at the plots
